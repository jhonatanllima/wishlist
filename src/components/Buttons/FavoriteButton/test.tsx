import { render, screen } from '@testing-library/react';

import { FavoriteButton } from '.';

describe('< FavoriteButton /> ', () => {
  it('should render the heading', () => {
    const { container } = render(<FavoriteButton />);

    expect(
      screen.getByRole('heading', {
        name: /FavoriteButton/i,
      })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
