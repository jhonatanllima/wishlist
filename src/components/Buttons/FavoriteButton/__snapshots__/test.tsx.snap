// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< FavoriteButton />  should render the favorite Button 1`] = `
.c0 {
  position: relative;
  width: 3rem;
  height: 4rem;
  border: none;
  border-radius: 0.3rem;
  background: #000000;
}

.c0:hover {
  cursor: pointer;
}

.c0:after {
  left: 0;
  bottom: 0;
  position: absolute;
  width: 0;
  height: 0;
  content: '';
  border-left: 1.5rem solid transparent;
  border-right: 1.6rem solid transparent;
  border-bottom: 0.7rem solid #FFFFFF;
}

<div>
  <button
    aria-label="Favoritar"
    class="c0"
  >
    <svg
      color="#FFFFFF"
      fill="currentColor"
      height="24"
      stroke="currentColor"
      stroke-width="0"
      style="color: rgb(255, 255, 255);"
      viewBox="0 0 512 512"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M462.3 62.6C407.5 15.9 326 24.3 275.7 76.2L256 96.5l-19.7-20.3C186.1 24.3 104.5 15.9 49.7 62.6c-62.8 53.6-66.1 149.8-9.9 207.9l193.5 199.8c12.5 12.9 32.8 12.9 45.3 0l193.5-199.8c56.3-58.1 53-154.3-9.8-207.9z"
      />
    </svg>
  </button>
</div>
`;
